Regular-Question-Answers/Base-7.cpp

```diff
@@ -0,0 +1,28 @@
+class Solution {
+public:
+    string convertToBase7(int num) {
+        // Returning number to positive if number is negative. 
```

Comment:
i think we need to rephrase this part because we are not checking whether or not the number is negative, we are just taking its absolute value regardless of the sign. maybe we could say something like "Taking the absolute value of num because we will operate on its distance to 0 and then convert it to its actual value, which can be positive or negative."